src/main/java/com/mongodb/shardsync/command/sync/SyncIndexesCommand.java

```diff
@@ -0,0 +1,50 @@
+package com.mongodb.shardsync.command.sync;
+
+import com.mongodb.shardsync.ShardConfigSync;
+import com.mongodb.shardsync.SyncConfiguration;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+import java.util.concurrent.Callable;
+
+@Command(name = "indexes", description = "Copy indexes from source to destination")
+public class SyncIndexesCommand implements Callable<Integer> {
+    
+    @CommandLine.ParentCommand
+    private SyncCommand parent;
+    
+    @Option(names = {"--collation"}, 
```

Comment:
Is this to make destination indexes use a different collation from the source? (Is there a need for that??)

Reply:
Yeah, I added this option to override the collation configured on the source. It was a pretty odd/rare scenario, so I've removed the option for clarity/simplicity.